templates/index.html

```diff
@@ -6,12 +6,16 @@
     <script async src="https://cdn.ampproject.org/v0.js"></script>
     <script async custom-element="amp-sidebar" src="https://cdn.ampproject.org/v0/amp-sidebar-0.1.js"></script>
     <script async custom-element="amp-selector" src="https://cdn.ampproject.org/v0/amp-selector-0.1.js"></script>
+    <script async custom-element="amp-consent" src="https://cdn.ampproject.org/v0/amp-consent-0.1.js"></script>
+    <script async custom-element="amp-geo" src="https://cdn.ampproject.org/v0/amp-geo-0.1.js"></script>    
     {{> head.html}}
     <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <meta name="amp-google-client-id-api" content="googleanalytics">
     <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
     <style amp-custom>
       {{> css/ampstart.css}}
-      {{> css/shared.css }}
+      {{> css/shared.css}}
+      {{> css/consent.css}}
```

Comment:
Can you inject this into shared.css instead - this way it'll be available to all pages.

Reply:
I can simply copy the CSS into shared.css, but I feel a little guilty, since we've then got a separate HTML file, but no corresponding separate CSS file.

I guess one alternative is to create a CSS build process, which is probably outside the scope of this PR... another is to make a CSS file that simply imports all of these... or I can just do what you suggest and we can deal with the problem if we ever get even more CSS files :)

Reply:
Sorry, I meant link it from inside `shared.css`:

```
{{> css/consent.css }}
```

Reply:
Hmm... this didn't work for me. 🙄 I don't know your build process that well - but when I tried this, in dist/, I still see that include in `shared.css`.  But the include never happens.  Instead, that include string vanishes at some point in the build process - perhaps during minification, or something.

I imagined that the include string might have been replaced by the file contents at some point in the build, but, for me, it doesn't. 😿

Reply:
tbh, I don't understand it either - let's keep it separate similar to ampstart.css etc. Sorry for the confusion.

Reply:
Ok! Reverted to separate version.

Reply:
Can you also delete the two imports from index.html

Reply:
Yes!  I had, but I hadn't committed the new file yet, pending other changes.  Will do so now.